build.cake

```diff
@@ -0,0 +1,91 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+var target = Argument("target", "Default");
+
+var sourceDir = "./exercises";
+var buildDir  = "./build";
+
+var defaultSolutionPath     = buildDir + "/Exercises.Default.sln";
+var refactoringSolutionPath = buildDir + "/Exercises.Refactoring.sln";
+
+var parallelOptions = new ParallelOptions { MaxDegreeOfParallelism = System.Environment.ProcessorCount };
+
+Task("Clean")
+    .Does(() => {
+		CleanDirectory(buildDir);   
+    });
+
+// Copy everything to build so we make no changes in the actual files.
+Task("CopyExercises")
+    .IsDependentOn("Clean")
+    .Does(() => {
+        CopyDirectory(sourceDir, buildDir);
+    });
+
+Task("EnableAllTests")
+    .IsDependentOn("CopyExercises")
+    .Does(() => {
+        var skipRegex = new Regex(@"(\[<Ignore\(""Remove to run test""\)>\]|, Ignore = ""Remove to run test case"")", RegexOptions.Compiled);
+        var testFiles = GetFiles(buildDir + "/*/*Test.fs");
+
+        foreach (var testFile in testFiles) {
+            var contents = System.IO.File.ReadAllText(testFile.FullPath);
+
+            if (skipRegex.IsMatch(contents)) {
+                var updatedContents = skipRegex.Replace(contents, "");
+                System.IO.File.WriteAllText(testFile.FullPath, updatedContents);
+            }
+        }
+    });
+
+Task("TestRefactoringProjects")
+    .IsDependentOn("EnableAllTests")
+    .Does(() => {
+        var refactoringSolution = ParseSolution(refactoringSolutionPath);
+        Parallel.ForEach(refactoringSolution.Projects, parallelOptions, (project) => DotNetCoreTest(project.Path.FullPath));
```

Comment:
Do you need the `noBuild` configuration setting like in the C# track?
https://github.com/exercism/csharp/pull/387/files

Reply:
No I don't, as the .NET CLI version 2.0 will automatically restore packages when the `dotnet test` command runs. In the C# track, we use the older version of the CLI which requires an explicit `dotnet restore` before that.

Reply:
Ah nice!